Makefile

```diff
@@ -23,12 +23,13 @@
 #   make remake [PL] [PR] [SD] [DF] - Cleans and rebuilds the target project (does not force-rebuild dependencies)
 #   make test [PL] [PR|LI] [TE] [SD] [DF] - Builds and runs the specified unit test, assuming all tests if TE is not defined
 #   make pytest [PR] [TE] - Runs the specified python unit test, assuming all tests in scripts directory of a project if TE is not defined
-#   make pytest_all - Runs all python tests in the scripts directory of every project 
+#   make pytest_all - Runs all python tests in the scripts directory of every project
```

Comment:
There seems to be a lot of these strange formatting changes in this file. Were these changes made by our formatter?

Reply:
yea this was made by running the make clean function i think. I was using that because i changed something in that function.

Reply:
probably your IDE removing trailing spaces, I think it's fine